File: code/Java_Class/week2/assignment2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in); 


        System.out.println("Please enter your name:");
        String name = s.nextLine();

        System.out.println("Enter your height in feet:");
        int feet = Integer.parseInt(s.nextLine());

        System.out.println("Enter your height in inches:");
        int inches = Integer.parseInt(s.nextLine());

        System.out.println("Enter your weight in pounds:");
        float pounds = Float.parseFloat(s.nextLine());

        float heightInInches = feet * 12 + inches;
        float bmi = pounds / (heightInInches * heightInInches) * 703;

        System.out.println("Hello " + name + ", your BMI is " + bmi);

    }
}

File: C++/Class_Work/week2/assignment2.cpp
#include <iostream>



int main () {
  std::string name;
  int feet;
  int inches;
  float pounds;
  std::cout << "please enter your name, height in feet, height in inches, and weight in pounds. press enter after each input." << "\n";
  std::cin >> name >> feet >> inches >> pounds;
  float bmi = pounds / ((feet * 12 + inches) * (feet * 12 + inches)) * 703;
  std::cout << "hello " << name << " your bmi is " << bmi << "\n";
  return 0;
}
